metrics/build.gradle

```diff
@@ -0,0 +1,11 @@
+description = 'OpenCensus Metrics Data Model'
```

Comment:
I think "data model" is more descriptive of the metrics Protocol Buffers, so it might be better to just call this artifact "OpenCensus Metrics".

Reply:
Make sense, done.